### ISSUE TITLE
Add a vignette introducing  group sequential design simulations by crafting from scratch

### ISSUE THREAD

**Issue by LittleBeannie** [Merck/simtrial#291]
This issue will add a vignette introducing group sequential design simulations by crafting from scratch.

- [x] Custom Group Sequential Design Simulations: Crafting from Scratch

**Comment by LittleBeannie** [comment]
This issue will be addressed after #255.

**Comment by LittleBeannie** [comment]
@keaven has 2 similar blog posts, and we save it as low priority.

**Comment by LittleBeannie** [comment]
This issue duplicates content covered in Keaven’s blog post: https://keaven.github.io/blog/delayed-effect-simulation/. Therefore, I will close this issue for now. We can reopen it if needed in the future.
